backend/apps/user/urls.py

```diff
@@ -0,0 +1,39 @@
+from rest_framework_simplejwt import views as jwt_view
+from rest_framework import routers
+from django.urls import path
+
+from .apis_auth import PasswordLogin, Signup, PasswordChangeOtp
+from .apis import ProfileViewSet
+
+# from apps.user.views import *
+
+
+USER_ULRS = {
+
+    'login_password': 'login/password/',
+    'change_password': 'password/change/',
+    'change_password_otp': 'password/change/otp/',
+    'set_password': 'password/set/',
+
+    'signup_api': 'signup/',
+    'profile_id': 'profile/<int:pk>/',
+    'profile': 'profile/',
+
+}
+
+router = routers.DefaultRouter()
+router.register('profile', ProfileViewSet, basename='profile')
+
+
+urlpatterns = [
+    path('token/refresh/', jwt_view.token_refresh),
```

Comment:
I think we can add the first three URLs to the `USER_ULRS` dictionary too.

Reply:
Those URLs and views are coming from jwt lib, not from business logics written by us. Also, there won' t be any changes in normal circumstances. That's why kept separately.